src/main/scala/nl.knaw.dans.easy.managedeposit/package.scala

```diff
@@ -35,11 +35,14 @@ package object managedeposit {
   val XML_NAMESPACE_XSI = "http://www.w3.org/2001/XMLSchema-instance"
   val XML_NAMESPACE_ID_TYPE = "http://easy.dans.knaw.nl/schemas/vocab/identifier-type/"
   val dateTimeFormatter: DateTimeFormatter = ISODateTimeFormat.dateTime()
+  val dateFormatter: DateTimeFormatter = ISODateTimeFormat.date()
   val notAvailable = "n/a"
 
   case class NotReadableException(path: Path, cause: Throwable = null)
     extends Exception(s"""cannot read $path""", cause)
 
+  case class DeleteParameters(filterOnDepositor: Option[DepositorId], age: Int, state: State.State, onlyData: Boolean, doUpdate: Boolean, newStateLabel: String = "", newStateDescription: String = "", output: Boolean = false)
```

Comment:
as discussed... suggestion:

```suggestion
  case class DeleteParameters(filterOnDepositor: Option[DepositorId], age: Int, state: State.State, onlyData: Boolean, doUpdate: Boolean, newState: Option[(State, String)] = None, output: Boolean = false)
```

Reply:
done